.github/workflows/todo.yaml

```diff
@@ -1,6 +1,9 @@
 name: todo
 on:
-- push
+  pull_request:
```

Comment:
Would this cause TODO issues to be created on every PR push that includes a new/different TODO comment?

Reply:
Hmm maybe the pull_request trigger is the wrong one here. I just wanted to limit this action running on tag events in my fork given our dev model is merging PRs to the main branch right now. I'll update any remove the pull_request field here.